masq_lib/src/messages.rs

```diff
@@ -501,9 +500,54 @@ pub struct UiConfigurationResponse {
     pub consuming_wallet_address_opt: Option<String>,
     #[serde(rename = "pastNeighbors")]
     pub past_neighbors: Vec<String>,
+    #[serde(rename = "paymentCurves")]
+    pub payment_curves: UiPaymentCurves,
+    #[serde(rename = "ratePack")]
+    pub rate_pack: UiRatePack,
+    #[serde(rename = "scanIntervals")]
+    pub scan_intervals: UiScanIntervals,
 }
```

Comment:
I propose we change the parameter set for the 'curves' to:

Payment Curves => Payment Thresholds